### PR TITLE
Tutorial1 readme

### PR DESCRIPTION
**Pull request by 5p0rt5BEArD**
# What

The room 1 tutorial written out as a step by step guide.
Linking to this and other tutorials from main readme.

The intention is for the style to be used across all tutorial readme files.
The intention is that a video can be recorded following the steps.

Video for this is here:
https://drive.google.com/file/d/1C6Zou1v_ad8OlvZjKmixuvBBLpVJvkRZ/view?usp=drive_link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +79,7 @@ You will need to enter your BIOS and activate Hardware Virtualisation. This is u
 </details>
 </details>
 
+<h3>Advanced options for builders</h3>
```
Comment (chrisfarms) [inline README.md:82]: maybe "contributors" would make it clearer this is mostly for people changing the game itself

```suggestion
<h3>Advanced options for contributors</h3>
```

File: contracts/src/maps/tutorial-room-1/README.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+- [Docker Desktop](https://docs.docker.com/get-docker/)
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
+# 1. Deploy the game
+
+First, deploy an instance of Downstream locally using Docker.
+
+- From a terminal in the repository root run:
+- `docker compose up --pull=always`
+
```
Comment (chrisfarms) [inline contracts/src/maps/tutorial-room-1/README.md:21]: might be good practice to use full code blocks wherever possible. This makes it look more like terminal entry, and allows picking a syntax.

````suggestion

From a terminal in the repository root run:

```bash
docker compose up --pull=always
```

````

File: contracts/src/maps/tutorial-room-1/README.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+# 4. Deploy the new tiles
+
+We will use the Downstream CLI to deploy our newly created tiles to our local Downstream instance.
+
+First, install the CLI
+- From a terminal run
+- `npm i -g @playmint/ds-cli`
+- Check it is installed from any terminal folder by running
+- `ds help`
+
```
Comment (chrisfarms) [inline contracts/src/maps/tutorial-room-1/README.md:80]:
````suggestion

First, install the CLI. From a terminal run:

```bash
npm i -g @playmint/ds-cli
```

Check it is installed from any terminal folder by running:

```bash
ds help
```

````

File: contracts/src/maps/tutorial-room-1/README.md [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+<img src="./readme-images/step4b.png" width=200>
+
+- Highlight and copy the key.
+
+Finally, run the command to deploy your map:
+- In a terminal at your new folder run the following
+- (where `<private-key>` can be pasted from the one you just copied)
+- `ds apply -n local -k <private-key> -f Locations.yaml`
+
```
Comment (chrisfarms) [inline contracts/src/maps/tutorial-room-1/README.md:97]:
````suggestion

Finally, run the command to deploy your map:

In a terminal at your new folder run the following, where `<private-key>` can be pasted from the one you just copied:

```bash
ds apply -n local -k <private-key> -f Locations.yaml
```

````

File: contracts/src/maps/tutorial-room-1/README.md [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+Browse to [localhost:3000]([http://localhost:3000]) and you should see your newly created map and be able to move your Unit around it!
+
+<img src="./readme-images/step4d.png" width=200>
+
+
+
+# 5. Create a new type of building
+
+The map is currently empty and there are no buildings to build. So we are going to create a new type of 'Factory' building that crafts a new kind of item.
```
Comment (chrisfarms) [inline contracts/src/maps/tutorial-room-1/README.md:110]: try and drum the word Kind in there heh

```suggestion
The map is currently empty and there are no buildings to build. So we are going to create a new kind of 'Factory' building that crafts a new kind of item.
```

File: contracts/src/maps/tutorial-room-1/README.md [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+To complete the tutorial, we're going to deploy your whole map folder together.
+
+First, stop the Downstream instance running in Docker with **ctrl-c**, and restart it by running `docker compose up`.
+
+Once you see "ready", browse to the game and check you have a single tile world.
+
+Now you can apply the whole map folder by passing the -R command and a folder path **ds apply**
+- At a terminal In your map folder run:
+- `ds apply -n local -k <private-key> -R -f .`
+
```
Comment (chrisfarms) [inline contracts/src/maps/tutorial-room-1/README.md:193]:
````suggestion
Now you can `ds apply` the whole map folder by passing the `-R` flag and the folder path. At a terminal In your map folder run:

```bash
ds apply -n local -k <private-key> -R -f .
```

````

**Review by chrisfarms — APPROVED** [review]
Left a few comments around preferring full code blocks over inline.

I think generally the use of bullet points can probably be reduced, since the entire document is kind of a sequential, so they may just be adding visual noise in some cases

File: contracts/src/maps/tutorial-room-1/README.md [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+- Choose a name.
+
+You can change the appearance of the building by clicking on the arrows and colour squares.
+
+<img src="./readme-images/step5b.png" width=200>
+
+Once you are happy, export the building source code:
+- Select **Export**.
+- Find the downloaded zip file. It will be named something like `BasicFactory.zip`.
+- Extract the files to the same folder as tiles.yaml.
```
Comment (hypnoshock) [inline contracts/src/maps/tutorial-room-1/README.md:131]: I think there isn't a `tiles.yaml` as one of the previous instructions were to rename the yaml to `Locations.yaml`
I think I read that correctly?

File: contracts/src/maps/tutorial-room-1/README.md [hunk 8]
```diff
@@ -0,0 +1,196 @@
+# Downstream Game Creation Tutorial 1
+## Fabricators and CLI
+
+We will follow the steps below to create a simple downstream map, with your own tile layout and your own factory building.
+
+Once complete, you will have used the core Downstream tools to create a map that looks like this when played:
+
+<img src="./readme-images/screenshot.png" width=300>
+
+# Prerequisites
```
Comment (chrisfarms) [inline contracts/src/maps/tutorial-room-1/README.md:10]: this heading and all subsequent should be H2s (there should only be one H1)

```suggestion
## Prerequisites
```

File: contracts/src/maps/tutorial-room-1/README.md [hunk 9]
```diff
@@ -0,0 +1,196 @@
+# Downstream Game Creation Tutorial 1
+## Fabricators and CLI
```
Comment (chrisfarms) [inline contracts/src/maps/tutorial-room-1/README.md:2]: should this first section be "aim" or "overview", something that can be consistent in each tutorial

```suggestion
# Downstream Game Creation Tutorial 1

## Aim
```

File: README.md (hunk 1 above)
Comment (5p0rt5BEArD) [reply README.md:82]: Agreed it should change from builders but I think Creators is better.

I think its bike shedding that matters in this case :)

Comment (5p0rt5BEArD) [reply README.md:82]: Oh hang on scrap that - this is the advance bit
Yes I like contributors !

Comment (chrisfarms) [reply README.md:82]: I'll continue the painting the shed  😆 .... I think "creators" still sounds too much like the "builder" group and not the set of people who may be committing changes back to this repository. Contributors is a github thing, and Contributor Guidelines, Contributor setup etc is something you will commonly see in github repos. 

Some "builders" may be or aspire to be "contributors" ... but what I was trying to put accross by using "contributor" was that the 90% of builder-type people can just gloss over that bit, and I think friendly names like "creators" don't make that clear enough.

Basically ... I don't want us to be supporting people trying to do `make dev` who don't know what Make is or have have a full Go toolchain on Windows and are just trying to spin up a local instance 😆

File: contracts/src/maps/tutorial-room-1/README.md (hunk 9 above)
Comment (5p0rt5BEArD) [reply contracts/src/maps/tutorial-room-1/README.md:2]: like it

File: README.md (hunk 1 above)
Comment (5p0rt5BEArD) [reply README.md]: Yep. I totally agree. And my original comment was my early morning brain thinking it was the title of the docker bit.

Sorry to make you do such a thorough paint job. Shed looks lovely now though.

File: contracts/src/maps/tutorial-room-1/README.md (hunk 2 above)
Comment (5p0rt5BEArD) [reply contracts/src/maps/tutorial-room-1/README.md]: Much nicer.
Done all of these as a new commit as I rebased before I could take your code suggestions.
